engines/neverhood/graphics.h

```diff
@@ -124,10 +124,10 @@ class BaseSurface {
 
 class ShadowSurface : public BaseSurface {
 public:
-	ShadowSurface(NeverhoodEngine *vm, int priority, int16 width, int16 height, BaseSurface *shadowSurface);
+	ShadowSurface(NeverhoodEngine *vm, int priority, int16 width, int16 height, const Common::SharedPtr<BaseSurface> &shadowSurface);
```

Comment:
Does it perhaps make sense to define a new `typedef` for brevity?

Reply:
Done